std_msgs/Header header
exploration/MarkerPositionState[] marker_states
